### DISCUSSION TITLE
朋友们，有一件重要的事哦！

### DISCUSSION THREAD

**Discussion by ybbgh** [share121/inter-knot#3240] [其他]
![180px-证物：「相互保证毁灭」按钮](https://github.com/user-attachments/assets/933ddf26-2327-45f0-bc25-28ae7f42d73e)
如果朋友们看见这个按钮，请按下他，在新艾利都各处都有哦。
如果按下的人足够，就会在新艾利都上空放一场绚丽的烟花哦！很期待吧！我也是哦！嘿嘿。
如果举报给治安局，打搅了这场盛宴...
晚上不要睡的太死哦。

**Comment by LT-UT** [comment]
第一个按下<img src="https://bbs-static.miyoushe.com/static/2024/06/19/8079e2c46c0698178c735198f9340cff_3596969066154449614.png" alt="花火 礼物" width="75" >

**Reply by ybbgh** [reply]
嘿嘿，谢谢啦

**Comment by qqqqqq77777** [comment]
六分街发现一个，按了<img src="https://bbs-static.miyoushe.com/static/2023/11/16/53069b115197426eff86537576a35d75_2306411376722926481.png" alt="外卖员邦布-出发" width="75" >

**Reply by ybbgh** [reply]
嘿嘿，谢了

**Comment by QingYu0x3f** [comment]
治安局桌上发现一个，按了

**Reply by ybbgh** [reply]
嘿嘿，谢了

**Comment by xjf0326** [comment]
发现一个按了<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜" width="75" >

**Reply by ybbgh** [reply]
嘿嘿，谢啦

**Comment by RaphaelHider** [comment]
新艾利都上空？那停着我的晖长石号呢，先等我开走你们再整，别炸到我的宝贝飞船。

**Reply by ybbgh** [reply]
走那么快干嘛，波及不到的<img src="https://bbs-static.miyoushe.com/static/2024/06/19/8079e2c46c0698178c735198f9340cff_3596969066154449614.png" alt="花火 礼物" width="75" >

**Comment by tayiie** [comment]
# 是哪个混蛋在报警按钮上放这个阿，我按了之后就被治安局的大屁股和小个子给拘了。
话说，那个大屁股的皮衣可真好看<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Reply by ybbgh** [reply]
你有些乐子啊，考虑加入我们假面愚者吗<img src="https://bbs-static.miyoushe.com/static/2024/01/31/cf527198a404a4db566dc4a9d68befc3_2947652818567379105.png" alt="花火 嘿嘿" width="75" >

**Reply by tayiie** [reply]
### 别介，这要是被花火老婆发现了，你们按完按钮后天上炸裂的不是烟花，而是我了。
> 花火大人，我对你可是忠心耿耿，放心吧，无论何时无论何地，不论你是否在关注着我，我对您玉足的忠心可是会像克里珀的巨锤一样永远不朽

**Reply by ybbgh** [reply]
呵呵呵，这么喜欢吗？加入我们每天都可以给你哦<img src="https://bbs-static.miyoushe.com/static/2024/01/31/cf527198a404a4db566dc4a9d68befc3_2947652818567379105.png" alt="花火 嘿嘿" width="75" >

**Comment by zxxiao111** [comment]
哲哥床上发现一个。按了

**Reply by ybbgh** [reply]
谢啦，嘿嘿

**Comment by wyqsagiri** [comment]
咖啡店看到一个，按了

**Reply by ybbgh** [reply]
嘿嘿，谢啦

**Comment by kaijinti** [comment]
真的吗？我在六分街捡到好几个都按了，也没啥反应啊

**Reply by ybbgh** [reply]
当然是真的啦！一定是数量还不够，到时候漫天花火绚丽绽放，想想就期待呢。谢谢啦
